src/provider/qti.js

```diff
@@ -276,7 +276,11 @@ var qtiProvider = {
             const feedbackPromise = new Promise(resolve => {
 
                 //@deprecated feedbacks from testContext
-                if (currentItem.hasFeedbacks || context.hasFeedbacks) {
+                if (
+                    (currentItem.hasFeedbacks || context.hasFeedbacks) &&
+                    context.remainingAttempts > 0 &&
+                    currentItem.remainingAttempts > 0
+                ) {
```

Comment:
nice catch, but  `not equal` instead of `equal`
![Screenshot from 2021-04-12 16-51-24](https://user-images.githubusercontent.com/8062729/114407315-f0b6e700-9bb0-11eb-9400-c22afa73e95f.png)